administrator/components/com_media/resources/scripts/store/mutation-types.js

```diff
@@ -40,3 +40,6 @@ export const HIDE_PREVIEW_MODAL = 'HIDE_PREVIEW_MODAL';
 export const SHOW_RENAME_MODAL = 'SHOW_RENAME_MODAL';
 export const HIDE_RENAME_MODAL = 'HIDE_RENAME_MODAL';
 export const RENAME_SUCCESS = 'RENAME_SUCCESS';
+
+// Search Query
+export const SEARCH_QUERY = 'SEARCH_QUERY';
```

Comment:
I see no mutation associated with this

Reply:
It would be nice if this mutation was named as

`export const SET_SEARCH_QUERY = 'SET_SEARCH_QUERY '` as it's self describing